course3/homework-string/my_string.cc

```diff
@@ -0,0 +1,105 @@
+#include <iostream>
+#include <cstring>
+#include "my_string.h"
+
+MyString::MyString(){
+	buffer = std::unique_ptr<char[]>(new char[1]);
+	buffer[0] = '\0';
+}
+
+MyString::MyString(const char* theString){
+	buffer = std::unique_ptr<char[]>(new char[std::strlen(theString) + 1]);
+	unsigned int i;
+	for (i = 0; i < std::strlen(theString); i++){
```

Comment:
wold be slightly simpler if you'd calculated the length once and the re-use it 3 times